src/components/parts/commons/atoms/Card/Card.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Box } from '@mui/system';
+import { Typography } from '../../atoms';
+import { Card } from './Card';
```

Comment:
エイリアスを設定しているので活用していきたいですね
```suggestion
import { Typography } from '~/components/atoms';
import { Card } from '~/components/atoms/Card';
```
みたいな感じです。メリットは階層に関係なくどこからでもそのファイルをインポートできることです